JASP-Engine/JASP/R/descriptives.R

```diff
@@ -291,7 +291,12 @@ Descriptives <- function(jaspResults, dataset, options) {
         shouldAddNominalTextFootnote      <- subReturn$shouldAddNominalTextFootnote
         shouldAddModeMoreThanOnceFootnote <- subReturn$shouldAddModeMoreThanOnceFootnote
         
-        stats$addRows(subReturn$resultsCol)
+        stats$addRows(subReturn$resultsCol, rowNames = paste0(variable, l))
+        
+        if (subReturn$shouldAddIdenticalFootnote)
+          stats$addFootnote(message = "All values are identical",
```

Comment:
```suggestion
          stats$addFootnote(message = gettext("All values are identical"),
```
to immediately add the translation :)